kvmagent/kvmagent/test/utils/snapshot_utils.py

```
@@ -59,3 +96,44 @@
+        "volumeUuid": vol_uuid,
+        "installPath": os.path.join(env.SNAPSHOT_DIR, '%s.qcow2' % uuidhelper.uuid()),
+        "vmInstanceUuid": vm_uuid,
+        "previousInstallPath": vol_path,
+        "snapshotUuid": uuidhelper.uuid(),
+        "memory": memory,
+        "live": True,
+        "full": False,
+        "volume": {
+            "volumeUuid": vol_uuid,
+            "installPath": vol_path,
+            "installPath": None,
```

Ruff (0.12.2)

Dictionary key literal `"installPath"` repeated

(F601)